### PR TITLE
8247564: Lanai - SwingSet2 - Motif L&F - UI controls border is incorrectly drawn with uiScale=1.0

### PR DESCRIPTION
**Pull request by dekonoplyov**
DrawLine implementation same as in OGLRenderer.c
It fixes JList borders and helps with lines inside radio buttons.

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [JDK-8247564](https://bugs.openjdk.java.net/browse/JDK-8247564): Lanai - SwingSet2 - Motif L&F - UI controls border is incorrectly drawn with uiScale=1.0 

### Reviewers
 * Ajit Ghaisas ([aghaisas](@aghaisas) - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/lanai pull/68/head:pull/68`
`$ git checkout pull/68`

### CODE REVIEW

File: src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m [hunk 1]
```diff
@@ -76,10 +76,63 @@ void MTLRenderer_DrawLine(MTLContext *mtlc, BMTLSDOps * dstOps, jint x1, jint y1
...
+        // horizontal
+        float fx1 = (float)x1;
+        float fx2 = (float)x2;
+        float fy  = ((float)y1) + 0.2f;
+
+        if (x1 > x2) {
+            float t = fx1; fx1 = fx2; fx2 = t;
+        }
+
+        struct Vertex v1 = {{fx1 + 0.2f, fy}};
```
Comment (aghaisas) [inline src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m]: We can avoid creation of v1 and v2 by directly assigning to verts[0] and verts[1].
8250778 changes can be referred.

Comment (dekonoplyov) [reply src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderer.m]: Fixed!

**Review by aghaisas — APPROVED** [review]

**Pull request closed without merging**